### ISSUE TITLE
error getting RDS binlog retention hours: Resource not found - after update to 1.12.3

### ISSUE THREAD

**Issue by kuzakov** [sbercloud-terraform/terraform-provider-sbercloud#316]
Hi there,

### Terraform Version
Terraform v1.9.5
on darwin_arm64

### Affected Resource(s)
- sbercloud_rds_instance

### Terraform Configuration Files
```hcl
esource "sbercloud_rds_instance" "rds_mysql_core" {
  name                  = "rds-mysql-core"
  region                = "ru-moscow-1"
  time_zone             = "UTC+03:00"
  enterprise_project_id = sbercloud_enterprise_project.project.id
  fixed_ip              = "x.x.x.x"
  flavor                = "rds.mysql.c2.large.ha"
  ha_replication_mode   = "async"
  security_group_id     = sbercloud_networking_secgroup.sg_rdsmysqlcore.id
  subnet_id             = sbercloud_vpc_subnet.core_main_subnet.id
  vpc_id                = sbercloud_vpc.vpc.id
  availability_zone     = ["ru-moscow-1c", "ru-moscow-1a"]
  tags = {
    "managed_by_terraform" = "true"
    }
  backup_strategy {
    keep_days  = 7
    start_time = "22:00-23:00"
  }

  db {
    port     = 3306
    type     = "MySQL"
    password = "blablapass"
    version  = "8.0"
  }
  volume {
    size = 40
    type = "ULTRAHIGH"
  }
  lifecycle {
    ignore_changes = [
      db,
    ]
  }
}
```

### Debug Output
Just only:
```
Error: error getting RDS binlog retention hours: Resource not found: [GET https://rds.ru-moscow-1.hc.sbercloud.ru/v3/10339484ae00272f2f2fc0081d3aa7f8/instances/79c773afc2b14a6087c93322bbdb8e2din01/binlog/clear-policy], request_id: 6f628952729b03abc482656792309425, error message: {"error_msg":"The API does not exist or has not been published in the environment","error_code":"APIGW.0101","request_id":"6f628952729b03abc482656792309425"}
│ 
│ 
│   with sbercloud_rds_instance.rds_mysql_core,
│   on 3_rds.tf line 1, in resource "sbercloud_rds_instance" "rds_mysql_core":
│    1: resource "sbercloud_rds_instance" "rds_mysql_core" {
```

### Expected Behavior
Will display the output of dry run and it provide opportunities to apply state

### Actual Behavior
Error

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
**This bug was began after update terraform provider from 1.12.0 to 1.12.3**

**Comment by 0ohmresistor** [comment]
Greetings!
This is a bug, we will fix it in the next release.

**Comment by 0ohmresistor** [comment]
We have fixed this in the new release 1.12.4
